src/components/configure/whiteListedDomain.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import { Form, Select, Input, Button, Alert } from 'antd';
+
+
+const WhiteListedDomains = ({ form }) => {
+    const { getFieldDecorator } = form;
+    const { Option } = Select;
+    const children = [];
+    for (let i = 10; i < 36; i++) {
```

Comment:
Why are you doing this? In whitelisted domains there wont be any options. User will just type and enter and it should be added as a chip